cfme/tests/automate/test_rest.py

```diff
@@ -29,3 +40,26 @@ def _do_query(**kwargs):
     filtered_depth = _do_query(depth='-1', search_options='state_machines')
     assert len(full_depth) > len(more_depth) > len(rest_api.collections.automate)
     assert len(full_depth) > len(filtered_depth) > 0
+
+
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.parametrize("method", ["POST", "DELETE"])
+def test_delete_automate_domain_from_detail(appliance, domain_rest, method):
```

Comment:
Optional: As far as I can tell, `test_delete_automate_domain` can be parametrized against `from_detail` and `from_collection`, though there may be some complication with `method`, so it may be more trouble than it's worth. I'll leave it as optional.

Reply:
Yep, more trouble than worth for! Actually, this is the pattern that is used throughout our code base, define 2 separte tests for deleting it via from_detail and from_collection. I could use `uncollectif` if I integrate these 2 tests, but this seems like the way to go!